runner/engine.go

```diff
@@ -251,7 +251,7 @@ func (e *Engine) watchPath(path string) error {
 				if excludeRegex {
 					break
 				}
-				if !e.isIncludeExt(ev.Name) {
+				if !e.isIncludeExt(ev.Name) && !e.checkIncludeFile(path) {
```

Comment:
I guess the argument here may need to be `ev.Name`, I will re-verify.

Reply:
I test the code below, it's ok:
```
if !e.isIncludeExt(ev.Name) && !e.checkIncludeFile(ev.Name) {
```

I don't know, why marked this pull request as draft, can i create new pull request base on the code.

Reply:
> I guess the argument here may need to be `ev.Name`, I will re-verify.

I test the code below, it's ok:
```
if !e.isIncludeExt(ev.Name) && !e.checkIncludeFile(ev.Name) {
```

I don't know, why marked this pull request as draft, can i recreate new pull request base on the code?